composer.json

```diff
@@ -12,10 +12,10 @@
     "require": {
         "php": ">=5.6.0",
         "guzzlehttp/guzzle": "~6.1",
-        "illuminate/console": "~5.0",
-        "illuminate/support": "~5.0",
-        "illuminate/routing": "~5.0",
-        "infusionsoft/php-sdk": "~1.3.6",
+        "illuminate/console": "~5.7",
+        "illuminate/support": "~5.7",
+        "illuminate/routing": "~5.7",
```

Comment:
@fkhateeb is there a reason why illuminate requirements was updated to `~5.7`?  I think the way we had it allowed for versions `>=5.0.0 but < 6.0.0`, now it is `>=5.7.0 and <6.0.0`. If you have good reason why this was updated, please let me know! Thanks!